qemu/tests/thin_write_in_qemu_img_commit.py

```diff
@@ -51,7 +51,13 @@ def _verify_map_output(output):
         qemu_version = env_process._get_qemu_version(qemu_path)
         match = re.search(r'[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?', qemu_version)
         host_qemu = match.group(0)
-        if host_qemu in VersionInterval('[6.1.0,)'):
+        if host_qemu in VersionInterval('[8.2.0,)'):
+            expected = {
+                "length": int(utils_numeric.normalize_data_size(
+                    params["write_size"], "B")),
+                "start": 0, "depth": 0, "present": True, "zero": True,
+                "data": False, "compressed": False}
```

Comment:
In L54-L65, The `expected` dict is almost the same, so I suggest defining the initial dict first and then inserting different keys based on QEMU version range.

So the initial can be `expected = {"length":int(utils_numeric.normalize_data_size(params["write_size"], "B")), "start": 0, "depth": 0, "zero": True, "data": False}`, QEMU 6.1.0-8.2.0, + `present`, > 8.2.0, + `compressed`.